challenge.md

```diff
@@ -5,20 +5,26 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone with the SSH URL
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+git diff
```

Comment:
We were looking for `git status`, but given how the question is worded, I can see why you put `git diff` - I won't take any points off. Really, you'll want to do both. `git status`  will show you all modified files, tracked and untracked, while `git diff` will show you specific changes in tracked files.